### DISCUSSION TITLE
更多的版本设置快捷方式

### DISCUSSION THREAD

**Discussion by E0x72-21** [Meloong-Git/PCL#5322] [提问]
能否在这个菜单下的快捷方式卡片中添加更多文件夹的快捷方式（截图文件夹，光影文件夹等）

![image](https://github.com/user-attachments/assets/a72df940-52e1-4005-9455-7ceafa8799a4)

**Comment by MoYuan-CN** [comment]
- #606

**Comment by LinQingYuu** [comment]
- 你是否在指 https://github.com/Hex-Dragon/PCL2/pull/4770
- 这个功能即将在 PCL2 的社区版可用 https://github.com/PCL-Community/PCL2-CE/pull/7 可以期待一下

**Reply by E0x72-21** [reply]
不懂就问，这PCL2 的社区版又是啥玩意

**Reply by E0x72-21** [reply]
而且我指的不是资源管理系统，我只是想要在版本设置下的快捷方式卡片里多加几个常用文件夹的快捷方式

**Reply by LinQingYuu** [reply]
> 而且我指的不是资源管理系统，我只是想要在版本设置下的快捷方式卡片里多加几个常用文件夹的快捷方式

这个管理系统实际上包含了打开各种文件夹的按钮，虽然不是在版本设置下的

> 不懂就问，这 PCL2 的社区版又是啥玩意

Look https://github.com/PCL-Community/PCL2-CE
